Frontend/src/App.tsx

```diff
@@ -1,10 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Login from './components/organisms/SignInCard/Login';
+import LoginPage from './components/pages/LoginPage';
+import SignUpPage from './components/pages/SignUpPage';
+import Dashboard from './components/pages/DashboardPage';
+import { Navigate, Route, Routes, useNavigate } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
+
 
 
 function App() {
+  const navigate=useNavigate();
+  const { user, isAuthenticated} = useAuth0();
+  const [accessToken, setAccessToken] = useState(
+    localStorage.getItem('accessToken')
+  )
```

Comment:
please don't store token in state. you should only use it when you are making an API call to a private endpoint that requires the token as part of request headers.